src/content/docs/en/extend/add-view-transitions.mdx

```diff
@@ -0,0 +1,434 @@
+
+      <details>
+      <summary>Show me the code.</summary>
+      ```astro title="src/components/Navigation.astro" ins='transition:animate="slide"'
+      ---
+      ---
+      <div transition:animate="slide" class="nav-links">
+        <a href="/">Home</a>
+        <a href="/about/">About</a>
+        <a href="/blog/">Blog</a>
+        <a href="/tags/" >Tags</a>
+      </div>
+      ```
+      </details>
+    </Box>
+
+    Check your browser preview and now both the page title and the header links will slide in on every page navigation.
+
+8. Add a longer fade-in on your blog post descriptions.
+
+    You can also customize Astro's built-in animations by importing them, then providing any CSS animation properties.
+
+    For example, to make the description fade in slowly when you navigate to a blog post, import the `fade` animation in your layout for Markdown blog posts. Then, add the transition directive for Astro's `fade` with a duration of `2s`:
+
+    ```astro title="src/layouts/MarkdownPostLayout.astro" ins={3} ins="transition:animate={fade({ duration: '2s' })}"
+    ---
+    import BaseLayout from "./BaseLayout.astro";
+    import { fade } from 'astro:transitions';
+    const { frontmatter } = Astro.props;
+    ---
+
+    <BaseLayout pageTitle={frontmatter.title}>
+      <p>{frontmatter.pubDate.slice(0, 10)}</p>
+      <p transition:animate={fade({ duration: '2s' })} ><em>{frontmatter.description}</em></p>
+      <p>Written by: {frontmatter.author}</p>
+      <img src={frontmatter.image.url} width="300" alt={frontmatter.image.alt} />
+      <slot />
+    </BaseLayout>
+    ```
+    Navigate to any blog post in your browser preview, and you will see the description fade in slower than the rest of the text.
+
+    📚 Read more about the different [transition directives](/en/guides/view-transitions/#transition-directives) and [customizing animations](/en/guides/view-transitions/#customizing-animations).
+
+### Force a full browser reload for some links
+
+9. Prevent client-side routing and instead require the browser to reload when navigating to your About page.
+
+    Sometimes you will want a full browser reload when visitors click a certain link. For example, you may be linking to a page that does not also use the `<ViewTransitions />` router, or to a file directly such as a `.pdf`. 
+
+    To make it so that your browser refreshes every time you click the navigation link to go to your About page, add the `data-astro-reload` attribute to the `<a>` tag in your `<Navigation/>` component. This will override the `<ViewTransitions />` router entirely, and any of the view transition animations, for this one link.
+
+     ```astro title="src/components/Navigation.astro" ins='data-astro-reload'
+      ---
+      ---
+      <div transition:animate="slide" class="nav-links">
+        <a href="/">Home</a>
+        <a href="/about/" data-astro-reload >About</a>
+        <a href="/blog/">Blog</a>
+        <a href="/tags/" >Tags</a>
```

Comment:
```suggestion
        <a href="/tags/">Tags</a>
```

Reply:
I am sorry that I am nitting spaces, but i figured with this feature it isn't too bad of a thing...?